app/models/live_result.rb

```diff
@@ -36,13 +38,65 @@ def potential_score
     complete? ? self[rank_by.to_sym] : best_possible_score
   end
 
+  def best_possible_score
+    1
+  end
```

Comment:
Why can't this be a constant, along the lines of `BEST_POSSIBLE_SCORE`?

Reply:
it can!